distribution/lib/Standard/Base/0.0.0-dev/src/Data/Json/Internal.enso

```diff
@@ -310,9 +312,9 @@ into_helper fmt json = case fmt of
     _ ->
```

Comment:
This `into_helper` may need to be revisited in some future - I think the way it (ab)uses the atoms to create the formats will be violating our up-coming type system.